Sources/Protocols/ErrorLogger.swift

```diff
@@ -0,0 +1,7 @@
+//  ErrorLogging.swift
+
+import Foundation
+
+public protocol ErrorLogger {
```

Comment:
Can you rename to Logger so it's easier to extend in the future? Header, filename and vars included.